src/services/asset-inventory/cloud-service/cloud-service-detail/modules/cloud-service-history-detail/CloudServiceHistoryChangesTab.vue

```diff
@@ -169,4 +179,24 @@ export default defineComponent({
         }
     }
 }
+
+@media (width < 478px) {
```

Comment:
모바일 이하 스타일을 지정할 땐 이렇게 합니다!
```suggestion
@screen mobile {
```